### PR TITLE
windows: Use fixed versions of msys packages

### PR DESCRIPTION
**Pull request by chrisforbes**

### CODE REVIEW

**Comment by hevrard** [comment]
maybe helpful: https://www.reddit.com/r/archlinux/comments/ep5mwj/system_update_without_zstd_support/

**Comment by hevrard** [comment]
For the sake or record, without a pacman system update, this command echoes 127 (foo.c and foo.patch are known to be valid):

`c:\tools\msys64\usr\bin\bash.exe --login -c "cd /t/src/github/agi/kokoro/windows/tmp ; /usr/bin/patch -p1 -i foo.patch ; echo $?"`

Now trying to get a proper system update.

File: kokoro/windows/build.bat [hunk 1]
```diff
@@ -20,39 +20,69 @@ set BUILD_ROOT=%cd%
...
+wget -q http://repo.msys2.org/msys/x86_64/libarchive-3.4.2-2-x86_64.pkg.tar.xz
+wget -q http://repo.msys2.org/msys/x86_64/libzstd-1.4.4-2-x86_64.pkg.tar.xz
+wget -q http://repo.msys2.org/msys/x86_64/zstd-1.4.4-2-x86_64.pkg.tar.xz
+wget -q http://repo.msys2.org/msys/x86_64/liblz4-1.9.2-1-x86_64.pkg.tar.xz
+wget -q http://repo.msys2.org/msys/x86_64/pacman-5.2.1-6-x86_64.pkg.tar.xz
+c:\tools\msys64\usr\bin\bash --login -c "pacman -U --noconfirm /t/src/zstd-1.4.4-2-x86_64.pkg.tar.xz /t/src/libzstd-1.4.4-2-x86_64.pkg.tar.xz /t/src/libarchive-3.4.2-2-x86_64.pkg.tar.xz /t/src/liblz4-1.9.2-1-x86_64.pkg.tar.xz /t/src/pacman-5.2.1-6-x86_64.pkg.tar.xz"
+c:\tools\msys64\usr\bin\bash --login -c "pacman -Syu -v --noconfirm"
+c:\tools\msys64\usr\bin\bash --login -c "pacman -Su -v --noconfirm"
+
+c:\tools\msys64\usr\bin\bash --login -c "echo HUGUES HERE SYSTEM UPDATE DONE"
```
Comment (pau-baiget) [inline kokoro/windows/build.bat:56]: Why this message?

Comment (chrisforbes) [reply kokoro/windows/build.bat]: Temporary hack, gone now.

**Pull request merged**